### ISSUE TITLE
Feature Request: Redo failed points at any later time later

### ISSUE THREAD

**Issue by Ruediger3** [mworion/MountWizzard4#171]
Hi Michel,

Current:
Missed points are repeated right after finishing the last point in the point model.

Feature:
To store the current status with the model to disk, and reload it with status so can repeat the failed points at any time and not only at right at this moment when making a new model.

Motivation:
When creating a model during full moon, there is a pretty large are you can not shot and solve. So you have a quite big undefined area. Repeating right after the last point is useless, since has not moved too far.
So it would be great to load the partial model after a couple of hours and repeat the missed points when the moon has significantly moved or is not visible at all. Also it would help to improve areas by adding some points, if it turns out there is a not well behaving area. 

What needs to be done:
Store the status of the model points after model run to file.  modify load and store. Have a button to add failed points to model.

Cheers
Rüdiger

**Comment by mworion** [comment]
Hi Rüdiger,
use case understood, but keeping a full state of model build stored in a full async behavior would stress the architectural design of MW4.
You anyway could combine multiple (partly) models afterwards (of course if you did not change mechanics in between).
What about offering an "auto-pause" before starting the retry ?
Michel

**Comment by Ruediger3** [comment]
Hi Michel,

I am not sure I understand the architectural conflict, since basically you only need to store the point spread and result for each point.

Combining two models could work, but never tried. The problem is, you have to create always a new specific secondary point spread for the failed area.  Probably it is easier and quicker to create a complete new model the next night. 

I think "auto-pause" is of no use in this case. You would have to wait at least one or two hours at minimum in the case of full moon. During that time you cannot use the system for any other observations or activities.

It a real asynchronous repeat of failed points is not feasible, the only option is to trash the model and make a new one.

BTW: I come to these points because I am currently collimating the CDK. Any adjustment voids the model significantly. So you have to redo the models very often. Also tonight where we have almost full moon. While preparing the best strategy I stumbled upon this use case. 

Cheers
Rüdiger

**Comment by mworion** [comment]
Rüdiger,
I understand your use case. Changing collimation (therefore I stay with refractors) invalidated model. Do you do only collimation work or do you in between use time for imaging as well ? If not the simplest way is really to take the model as is and redo it next time. Otherwise, let me think how we could sort this out without changing to much (and much work)
Michel

**Comment by Ruediger3** [comment]
The model is required, other wise you cannot slew to the bright stars to continue your work. If you work at extream large FL you have no chance to find your calibration star manually again. 
I have tried the last time and that took me to the "align model" issue. But if this is working now, it might be sufficient for this use case. 

Also after finishing you work, you want to use it. Hence you create a fresh model - as good as possible in that night. If you have a definition gap due to full moon, you have to re-do the model the next day instead of just filling the gap.
It is not crucial, but it safes a lot of time. 

Let's put this on hold. Maybe if we both get bored (which we both know will never happen :) ) we can review this point. 
I will check out how it is working tonight. Maybe I have a new idea. Let's see.

Anyway thanks for your great support.
